docs/settings/cases-settings.asciidoc

```diff
@@ -0,0 +1,15 @@
+[[cases-settings]]
+== Cases settings in {kib}
+++++
+<titleabbrev>Cases settings</titleabbrev>
+++++
+
+You do not need to configure any additional settings to use <<cases,cases>> in {kib}.
+To provide greater control over case features, you can configure the following settings in the `kibana.yml` file:
+
+`xpack.cases.files.allowedMimeTypes`::
+The MIME types that you can attach to a case, represented in an array of strings.
```

Comment:
I think it would be nice to provide a simple example of the necessary formatting, i.e. something like this:
```suggestion
The MIME types that you can attach to a case, represented in an array of strings. For example: `['image/tiff','text/csv','application/zip'].`
```